utils_linux.go

```diff
@@ -46,9 +47,33 @@ func loadFactory(context *cli.Context) (libcontainer.Factory, error) {
 	if rootlessCg {
 		cgroupManager = libcontainer.RootlessCgroupfs
 	}
+
+	var sp *configs.SystemdProperties
 	if context.GlobalBool("systemd-cgroup") {
 		if systemd.UseSystemd() {
 			cgroupManager = libcontainer.SystemdCgroups
+			if context.GlobalIsSet("systemd-property") {
+				sp = &configs.SystemdProperties{}
+				systemdProperties := context.GlobalStringSlice("systemd-property")
+				for _, prop := range systemdProperties {
+					pair := strings.Split(prop, "=")
+					if len(pair) != 2 {
+						return nil, fmt.Errorf("systemd-property %q not of the form key=pair", prop)
+					}
+					switch pair[0] {
+					// TODO: Reuse SupportedStemdProperties here when we add more properties
+					case "stop-timeout":
```

Comment:
Any reason why we would want to switch on these one by one and not support a generic map so that any additional properties can be added?  

```bash
runc --systemd-property TimeoutStopUSec=1000000 --systemd-property TimeoutStartUSec=10000000
``` 

Seems like if we are going to add this, might as well make it generic so we can support all of them.  I'd probably just `map[string]interface{}` as the `newProp` func already handles the interface and makes a varint with it